### DISCUSSION TITLE
Inbox notifications icons meaning

### DISCUSSION THREAD

**Discussion by in43sh** [community/community#119485] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I see icons next to the notifications in my inbox. Is there any legend or a page that explains all the icons meaning? I attached a screenshot as an example. On the screenshot, there are only two types of icons but I've seen other icons before.

![Screenshot 2024-04-14 125306](https://github.com/community/community/assets/3116876/6b9a94ec-d4ed-4079-85aa-232ec47da6f3)

**Comment by send0moka** [comment]
The icons in your screenshot typically represent:
- Green icon - Pull request
- Purple icon - Merge

**Reply by ebndev** [reply]
To add onto this, you can also hover over the icons and it will say what the icons represent 😀

**Reply by in43sh** [reply]
> The icons in your screenshot typically represent:

> Green icon - Pull request
> Purple icon - Merge

and red?

**Comment by in43sh** [comment]
> To add onto this, you can also hover over the icons and it will say what the icons represent 😀

It doesn't work for me. Can you share a screenshot?
